ReactAndroid/hermes-engine/build.gradle

```diff
@@ -71,6 +71,18 @@ task installArchives {
     dependsOn("publishAllPublicationsToNpmRepository")
 }
 
+def replaceContent = '''elseif(CMAKE_HOST_WIN32)
+    set (ABI_NORM \"\")
+    if (ANDROID_ABI STREQUAL \"armeabi-v7a\" )
+      set (ABI_NORM \"arm\")
+    elseif(ANDROID_ABI STREQUAL \"arm64-v8a\" )
+      set (ABI_NORM \"arm64\")
+    else()
+      set (ABI_NORM \"\\${ANDROID_ABI}\")
+    endif()
+    set( value \"\\${ABI_NORM}-unknown-linux-gnu\" )
+  else( MSVC )'''
```

Comment:
Can we just extend the file inside the Hermes Repo here?
https://github.com/facebook/hermes/blob/main/external/llvh/cmake/modules/GetHostTriple.cmake

Reply:
Yeah .. Here is the PR : https://github.com/facebook/hermes/pull/747